src/testdrive/src/action.rs

```diff
@@ -77,6 +76,8 @@ pub struct Config {
     pub reset_materialized: bool,
     /// Emit Buildkite-specific markup.
     pub ci_output: bool,
+    /// Default SQL timeout.
+    pub default_timeout: f64,
```

Comment:
Rust convention is to convert to more structured types ASAP, so I'd say use `Duration` here and convert from `f64` to `Duration` in `main.rs`.

Reply:
I need to check about that -- when I first tried this about 1 month ago, I observed that ```Duration``` was "consumable", in a sense that repeated uses of it to drive timeouts caused the interval to go down to zero. I will recheck that.